make/conf/jib-profiles.js

```diff
@@ -999,55 +999,37 @@ var getJibProfilesProfiles = function (input, common, data) {
         testOnlyProfilesPrebuilt["run-test-prebuilt"]["dependencies"].push(testedProfile + ".jdk_symbols");
     }
 
+    var testOnlyProfilesPrebuiltDocs = clone(testOnlyProfilesPrebuilt);
+    testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"] = testOnlyProfilesPrebuiltDocs["run-test-prebuilt"];
+    delete testOnlyProfilesPrebuiltDocs["run-test-prebuilt"];
```

Comment:
I think this could be simplified to something like (untested):
```suggestion
    var testOnlyProfilesPrebuiltDocs = {};
    testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"] = clone(testOnlyProfilesPrebuiltDocs["run-test-prebuilt"]);
```

Reply:
This exact code didn't work, but I now have a more concise version